sklearn/gaussian_process/gpr.py

```diff
@@ -128,6 +128,8 @@ def optimizer(obj_func, initial_theta, bounds):
     log_marginal_likelihood_value_ : float
         The log-marginal-likelihood of ``self.kernel_.theta``
 
+    rng_ : numpy.RandomState
+
```

Comment:
You  need to explain what this attribute is. Look at the above declarations. The user needs to know what this stands for.